src/main/java/ir/sahab/kafkaconsumer/SmartCommitKafkaConsumer.java

```diff
@@ -132,13 +131,19 @@
      * We have used a same callback that does nothing other than writing simple logs and resetting
      * the {@link #offsetTracker}.
```

Comment:
Replace: 
We have used ... -> It resets the {@link #offsetTracker} and also calls the rebalance listener provided by the client.